cli/tests/test_release_workflow_staged.py

```
@@ -811,7 +810,7 @@ def test_release_certificate_is_canonicalized_and_authenticated_before_seal() ->
     assert "scripts/release_candidate.py verify" in seal_script
 
 
-def test_exact_posix_fresh_install_and_twelve_upgrade_cells_gate_publication() -> None:
+def test_exact_posix_fresh_install_twelve_upgrades_and_intel_refusal_gate_publication() -> None:
```

Ruff (0.16.1)

[warning] Too many statements (53 > 50)

(PLR0915)